manifests/08-openshift-imagestreams.yaml

```diff
@@ -164,10 +180,12 @@ metadata:
     include.release.openshift.io/single-node-developer: "true"
     include.release.openshift.io/single-node-production-edge: "true"
 spec:
+  lookupPolicy:
+    local: true
```

Comment:
Likewise, openshift/hello-openshift resolves to a Docker Hub image.

Reply:
That is actually the entire point of the (new in 4.7) hello-openshift imagestream, to easily replace uses of the unmaintained, single-arch, and now rate-limited image on docker.io.